docs/DeveloperGuide.md

```diff
@@ -253,6 +253,39 @@ User Guide.
     * Cons: Lacks feedback on whether the persons were added, which may be useful for confirmation and may result in
       duplicate entries.
 
+### Edit feature
+
+#### Implementation
+
+The `edit` feature allows user to edit a person's data in the address book by providing the prefix and the changed value.
+
+Edit implements the following operations:
+* `EditCommand#execute`
+* `EditCommand#createEditedPerson`
+
+EditCommand also contains a static class `EditPersonDescriptor` that stores the details to edit the person with.
+
+Given below is an example usage scenario and how the clone mechanism behaves at each step.
```

Comment:
Was this intentional? Not sure if you are showing clone mechanism or edit mechanism

Reply:
thanks for noticing the mistake glenn!

Reply:
hehe LGTM! merging now